content.json

```diff
@@ -1,13 +1,13 @@
 {
- "address": "1MusicXkuN2pk5hRdmroeyaCLDJtzTicpB",
+ "address": "152qbnvTwEDHoDSecNQhYoR6z78QVLpE4f",
```

Comment:
I think you should not commit the modifications you did in `content.json` (I mean, the ones that are done by signing the zite), since if I merge it il will modify the publick key and the signatures on the official repo and I'll have to change it after.